internal/testhelpers/e2e/session.go

```diff
@@ -182,6 +182,13 @@ func new(t *testing.T, retainDirs, updatePath bool, extraEnv ...string) *Session
 	err = fileutils.Touch(filepath.Join(dirs.Base, installation.InstallDirMarker))
 	require.NoError(session.T, err)
 
+	cfg, err := config.New()
+	require.NoError(session.T, err)
+
+	if err := cfg.Set(constants.SecurityPromptConfig, false); err != nil {
```

Comment:
I just want to double-check that this is an okay place/way to configure something like this. I'm used to seeing environment variables used to define behavior like this, like in `sandboxedTestEnvironment()`.

Reply:
We could introduce an environment variable to configure this for testing purposes though that feels redundant with the config. I don't believe there should be any issues setting it this way as each config is contained to a single test, maybe a slight performance hit.